util/src/main/java/io/kubernetes/client/util/Config.java

```diff
@@ -107,11 +109,7 @@ public static ApiClient fromConfig(Reader input) {
         String caCert = config.getCertificateAuthorityData();
         String caCertFile = config.getCertificateAuthorityFile();
         if (caCert != null) {
-            try {
-                client.setSslCaCert(new ByteArrayInputStream(caCert.getBytes("UTF-8")));
-            } catch (UnsupportedEncodingException ex) {
-                ex.printStackTrace();
-            }
+            client.setSslCaCert(new ByteArrayInputStream(Base64.decodeBase64(caCert)));
```

Comment:
I think using `SSLUtils.getInputStreamFromDataOrFile` is a better solution. We can close this one and merge #27 or fix this one. @lwander